exercises/practice/swift-scheduling/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# Instructions Append
+
+```exercism/note
+`elm/time` is not the most complete package when it comes to handling date and time operations.
+In real world applications, one would reach out to packages from the Elm community, but since those are not accessible in the online editor, take this exercise as a challenge rather than as a practice of idiomatic Elm.
```

Comment:
```suggestion
In real world applications, one would reach out to packages from the Elm community to manipulate dates and times, but since those are not accessible in the online editor, take this exercise as a challenge rather than as a practice of idiomatic Elm.
```